rdr_service/config.py

```diff
@@ -85,6 +85,8 @@
 EHR_STATUS_BIGQUERY_VIEW_ORGANIZATION = "ehr_status_bigquery_view_organization"
 HPO_REPORT_CONFIG_MIXIN_PATH = "hpo_report_config_mixin_path"
 LOCALHOST_DEFAULT_BUCKET_NAME = 'local_bucket'
+BIOBANK_SAMPLES_INVENTORY_FILE_PATTERN = 'biobank_samples_inventory_file_pattern'
+BIOBANK_SAMPLES_INVENTORY_MANIFEST_FILE_PATTERN = 'biobank_samples_inventory_manifest_file_pattern'
```

Comment:
Would it help if these had more obvious names that reflect the daily vs. monthly inventory report purpose?   E.g., BIOBANK_SAMPLES_DAILY_INVENTORY_FILE_PATTERN / MONTLY_INVENTORY_MANIFEST_FILE_PATTERN?

Reply:
Good idea, updating it now.